### PR TITLE
Update dotnet-watch.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-5.0&branch=pr-en-us-19830#run-net-core-cli-commands-using-dotnet-watch)

### CODE REVIEW

File: aspnetcore/tutorials/dotnet-watch.md [hunk 1]
```diff
@@ -75,10 +75,18 @@ Any [.NET Core CLI command](/dotnet/core/tools#cli-commands) can be run with `do
 
 Run `dotnet watch run` in the *WebApp* folder. The console output indicates `watch` has started.
 
-::: moniker range=">= aspnetcore-3.1"
-Starting with the .NET Core 5.0 SDK, invoking `dotnet watch run` on a webapp launches a browser that navigates to the app's url once ready. `dotnet watch` does this by reading the app's console output and waiting for the the ready message printed by <xref:Microsoft.AspNetCore.WebHost>. 
+::: moniker range=">= aspnetcore-5.0"
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/dotnet-watch.md:78]: I changed this to 5 from >= 3.1

File: aspnetcore/tutorials/dotnet-watch.md [hunk 2]
```diff
@@ -184,15 +192,16 @@ dotnet watch msbuild /t:Test
...
 
 Some configuration options can be passed to `dotnet watch` through environment variables. The available variables are:
 
-| DOTNET_USE_POLLING_FILE_WATCHER                | If set to "1" or "true", `dotnet watch` will use a polling file watcher instead of CoreFx's `FileSystemWatcher`. Used when watching files on network shares or Docker mounted volumes.                       |
-| DOTNET_WATCH_SUPPRESS_MSBUILD_INCREMENTALISM   | By default, `dotnet watch` optimizes the build by avoiding certain operations such as running restore or re-evaluating the set of watched files on every file change. If set to "1" or "true",  these optimizations are disabled. |
-| DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER   | `dotnet watch run` will attempt to launch browsers for web apps with `launchBrowser` configured in `launchSettings.json`. If set to "1" or "true", this behavior is suppressed. |
-| DOTNET_WATCH_SUPPRESS_BROWSER_REFRESH   | `dotnet watch run` will attempt to refresh browsers when it detects file changes. If set to "1" or "true", this behavior is suppressed. This behavior is also suppressed if DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER is set. |
+| Setting  | Behavior |
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/dotnet-watch.md:199]: @pranavkm  suggest better column headings

**Comment by Rick-Anderson** [comment]
@pranavkm please review and suggest changes.

File: aspnetcore/tutorials/dotnet-watch.md (hunk 1 above)
Comment (pranavkm) [reply aspnetcore/tutorials/dotnet-watch.md:78]: Fair enough. Technically this works on 3.1 projects, but I'm ok not advertising it.

File: aspnetcore/tutorials/dotnet-watch.md (hunk 2 above)
Comment (pranavkm) [reply aspnetcore/tutorials/dotnet-watch.md:199]:
```suggestion
| Setting  | Description |
```

Perhaps?

**Review by pranavkm — APPROVED** [review]

**Pull request merged**